### DISCUSSION TITLE
WebView2 package

### DISCUSSION THREAD

**Discussion by gvanem** [sumatrapdfreader/sumatrapdf#3235] [Q&A]
I have a question on the use of the Microsoft WebView2 package in Sumatra?
When and how is it used? 

I note only a reference to it in the `-test-browser` command-line option when built with `DEBUG` (launching 
Krzysztof Kowalczyk home-page). What else does  WebView2 do besides with this option and with `DEBUG`?

**Comment by kjk** [comment]
It's not yet used. A potential use would be for .chm files (where we currently use the old web control) and maybe for displaying some UI elements. But I haven't yet finished implementing the control.
